lua/telescope/builtin/init.lua

```diff
@@ -25,6 +29,19 @@ local builtin = {}
 builtin.live_grep = require('telescope.builtin.files').live_grep
 
 builtin.grep_string = require('telescope.builtin.files').grep_string
+
+--- Lists files in current directory using `fd`, `ripgrep` or `find`.
+--- It will not show directories or gitignored files (when using `fd` or `ripgrep`).
+--- Later can be enabled with a different `find_command`.
+---@param opts table: Configure behavior of `find_files`.
+---@field find_command table: Specify your find command as a table of strings.
+---@field hidden boolean: Enable or disable hidden files. Default is false.
+---@field follow boolean: Enable or disable if symbolic links should be followed. Default is false.
+---@field cwd string: Specify the path on which find_files should run.
+---@field search_dirs table: Specify multiple search directories as table of paths.
+---@field entry_maker function: How entries are generated. Can be changed if you want a different display output.
+---@field disable_devicons boolen: Enable or disable devicons in front of each entry. Default is true if nvim-web-devicons is installed.
+---@field shorten_path boolean: Enable or disable if paths should be shortened. Default is false.
```

Comment:
CC @smithbm2316 @fdschmidt93

That would be an example of how we could document builtins :) multiline `@param`/`@field` is will be done soon.

Reply:
Thanks for the example! Sorry I've been a bit absent on this lately, I'm finishing up my last couple of days of my undergrad degree and I'll hopefully be back able to work on docs starting next week!

Reply:
Very nice! I'll also try to get started writing up docs on my end towards end of this week or beginning of next week :)